mne/io/edf/edf.py

```diff
@@ -605,6 +1111,9 @@ def read_raw_edf(input_fname, montage=None, eog=None, misc=None,
     verbose : bool, str, int, or None
         If not None, override default verbose level (see :func:`mne.verbose`
         and :ref:`Logging documentation <tut_logging>` for more).
+    biosemi : bool (default False)
+        Set to True to when reading Biosemi data as GDF files, otherwise the
+        stimulus channel will not be properly parsed.
```

Comment:
no, because you can have GDF files recorded from a biosemi system (through [`biosemi2ft`](http://www.fieldtriptoolbox.org/development/realtime/biosemi) for instance)